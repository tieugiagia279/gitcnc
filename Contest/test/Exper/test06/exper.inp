50
18828 94
91025 6247
66532 5539
37732 4245
69285 3638
43870 495
69228 1541
46116 373
62102 8085
9422 5465
57665 871
24993 4426
16385 1487
14982 7208
54580 4955
48628 5001
35209 2001
99500 729
86498 5240
98540 7449
15574 2874
62602 4493
9322 5987
3155 3697
39620 8046
44864 564
15168 7685
8705 7700
47083 8381
39334 369
20448 1101
81372 6514
89011 8526
34304 6081
19406 4069
69959 6273
43838 8734
76336 1372
61471 2419
81269 2802
12269 3054
53555 8630
58148 8330
89617 464
24222 706
82338 7242
57349 1050
22036 5098
87887 3894
78000 7501
