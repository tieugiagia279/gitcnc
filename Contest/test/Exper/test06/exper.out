4
3
5
4
3
3
2
3
8
4
4
5
4
5
4
2
3
1
3
3
5
2
5
1
9
4
14
7
6
2
2
3
11
6
6
5
9
2
2
2
5
10
8
2
2
4
4
8
2
9
